apps/src/code-studio/initApp/project.js

```diff
@@ -973,6 +973,14 @@ var projects = module.exports = {
   },
 
   /**
+   * Returns the URL stored in current.thumbnailUrl.
+   * @returns {string} The thumbnail URL.
+   */
+  getThumbnailUrl() {
+    return current.thumbnailUrl;
```

Comment:
to protect against `current` not having been initialized, please change this to: `return current && current.thumbnailUrl;`